portfolio/src/main/webapp/script.js

```diff
@@ -72,21 +87,23 @@ function getComments() {
 
 /* Creates an <li> element containing text. */
 function addComment(comment) {
-  const commentItem = document.createElement("li");
-  const commentHeader = document.createElement("div");
-  const commentName = document.createElement("p");
-  const commentMessage = document.createElement("p");
-  const commentDelete = document.createElement("img");
-
-  commentItem.classList.add("comment");
-  commentHeader.classList.add("comment-header");
-  commentName.classList.add("comment-name");
-  commentMessage.classList.add("comment-message");
-  commentDelete.classList.add("comment-delete");
-
-  commentName.innerText = comment.name;
-  commentMessage.innerText = comment.message;
-  commentDelete.onclick = function() { deleteComment(comment.id); }; 
+  const commentItem = createElementWithParams("li", { className: "comment" })
+  const commentHeader = createElementWithParams("div", { className: "comment-header" })
+
+  const commentName = createElementWithParams("p", { 
+    className: "comment-name", 
+    innerText: comment.name 
+  })
+
+  const commentMessage = createElementWithParams("p", { 
+    className: "comment-message", 
+    innerText: comment.message 
+  })
+
+  const commentDelete = createElementWithParams("img", { 
+    className: "comment-delete", 
+    event: { "click": () => deleteComment(comment.id) } 
```

Comment:
I'm surprised this works. I expected that this needed to be

```
event: () => deleteComment(comment.id),
```

on this line.